packages/styles/src/elements/paragraph.scss

```diff
@@ -0,0 +1,9 @@
+@use 'sass:map';
+@use '../functions/tokens';
+@use '../tokens/elements';
+
+tokens.$default-map: elements.$post-paragraph;
+
+p {
+  margin-block-start: tokens.get('paragraph', 'margin-block-start');
```

Comment:
I think we need to reset the margin-block-end here (is there no token for it?).

Reply:
Currently, there is not token for the ` margin-block-end`.

Reply:
Isn't the margin-block-end padding reseted through the reset.scss?
At least there is an active rule: `* { margin: 0; }` present in the browser.

Reply:
```suggestion
  margin-block-start: tokens.get('paragraph', 'margin-block-start');
  
  &:first-child {
    margin-block-start: 0;
  }
```